src/sentenceCount.ts

```diff
@@ -0,0 +1,8 @@
+const sentence = 'Esta es una frase de prueba, que tiene 10 palabras';
+
+function wordsCount(): number {
+  const sentenceArray = sentence.split(' ');
+  return sentenceArray.length;
+}
+
+console.log(`The sentence '${sentence}' has ${wordsCount()} words`);
```

Comment:
```suggestion
console.log(`The sentence '${sentence}' has ${wordsCount()} words`);

```

Para evitar este tipo de problemas normalmente se pone una línea en blanco al final del fichero

![CleanShot 2021-11-10 at 19 26 54@2x](https://user-images.githubusercontent.com/2389019/141180087-c1cfab4e-c3ce-48eb-b7ba-dc4121d908a3.png)